example/App.tsx

```diff
@@ -12,12 +12,13 @@ import { icons } from './App.preset';
 import { styles } from './App.styles';
 import { Button } from './src/Button';
 import { IconButton } from './src/IconButton';
+import { AlternateAppIcons } from 'expo-alternate-app-icons/AlternateAppIconsType';
```

Comment:
Let's export type `AlternateAppIcons` directly from the index file to avoid importing from sub file.